src/SwedbankPay.Sdk/PaymentInstruments/Swish/SwishPaymentRequest.cs

```diff
@@ -1,28 +1,28 @@
-﻿using System.Collections.Generic;
-using System.Globalization;
-
-namespace SwedbankPay.Sdk.Payments.SwishPayments
-{
-    public class SwishPaymentRequest
-    {
-        public SwishPaymentRequest(CurrencyCode currency,
-                              List<Price> prices,
-                              string description,
-                              string payerReference,
-                              string userAgent,
-                              CultureInfo language,
-                              Urls urls,
-                              PayeeInfo payeeInfo,
-                              PrefillInfo prefillInfo,
-                              bool isEnabledForEcommerceOnly = false,
-                              Dictionary<string, object> metadata = null)
-        {
-            var swishRequest = new SwishPaymentOptionsObject(isEnabledForEcommerceOnly);
-            Payment = new SwishPaymentRequestObject(currency, prices, description, payerReference, userAgent, language, urls, payeeInfo,
-                                               prefillInfo, swishRequest, metadata);
-        }
-
-
-        public SwishPaymentRequestObject Payment { get; }
-    }
+﻿using System.Collections.Generic;
+using System.Globalization;
+
+namespace SwedbankPay.Sdk.Payments.SwishPayments
+{
+    public class SwishPaymentRequest
+    {
+        public SwishPaymentRequest(CurrencyCode currency,
+                              List<Price> prices,
+                              string description,
+                              string payerReference,
+                              string userAgent,
+                              CultureInfo language,
+                              Urls urls,
+                              PayeeInfo payeeInfo,
+                              PrefillInfo prefillInfo,
+                              bool isEnabledForEcommerceOnly = false,

```

Comment:
Codacy found an issue: [Use the overloading mechanism instead of the optional parameters.](https://app.codacy.com/gh/SwedbankPay/swedbank-pay-sdk-dotnet/pullRequest?prid=6290774)